radio/src/targets/common/arm/stm32/rotary_encoder_driver.cpp

```diff
@@ -104,7 +104,11 @@ void rotaryEncoderCheck()
 #else
   uint8_t newPosition = ROTARY_ENCODER_POSITION();
   if (newPosition != rotencPosition && !(readKeys() & (1 << KEY_ENTER))) {
-    if ((rotencPosition & 0x01) ^ ((newPosition & 0x02) >> 1)) {
+#if defined(RADIO_ZORRO)
```

Comment:
Can we have this a bit more "generic"? Like "INVERTED_ROTARY_ENCODER", or something like that?

Reply:
OK, changed, please have a look if this is more in the vain you had in mind?